tachyon/math/finite_fields/generator/prime_field_generator/build_defs.bzl

```diff
@@ -297,12 +306,17 @@ def _do_generate_prime_fields(
             **kwargs
         )
 
+    if is_small_prime_field:
+        prime_field_gpu_dep = small_prime_field_dep
+    else:
+        prime_field_gpu_dep = "//tachyon/math/finite_fields:prime_field_gpu"
+
     tachyon_cc_library(
         name = "{}_gpu".format(name),
         hdrs = [":{}_gen_gpu_hdr".format(name)],
         deps = [
             ":{}_config".format(name),
-            "//tachyon/math/finite_fields:prime_field_gpu",
+            prime_field_gpu_dep,
```

Comment:
When binary field is introduced, the code should be change as follows: 

https://github.com/kroma-network/tachyon/blob/4c639761dbe101cab3378688806cafa0be0982d3/tachyon/math/finite_fields/generator/prime_field_generator/build_defs.bzl#L374-L389

```bazel
    if is_binary_prime_field:
        prime_field_gpu_dep = binary_prime_field_dep
    elif is_small_prime_field:
        prime_field_gpu_dep = small_prime_field_dep
    else:
        prime_field_gpu_dep = "//tachyon/math/finite_fields:prime_field_gpu"

    tachyon_cc_library(
        name = "{}_gpu".format(name),
        hdrs = [":{}_gen_gpu_hdr".format(name)],
        deps = [
            ":{}_config".format(name),
            prime_field_gpu_dep,
        ],
        **kwargs
    )
```

How do you think then?

Reply:
Though I'm not sure on what the difference is when binary field is introduced, I still think a ternary operator may be cleaner, especially since  some if/else statements using `is_small_prime_field` are stated before, so it feels like this if/else is possibly redundant as well.

Reply:
In the case that the if statements include more than 2 branches, a ternary operator should not be used, so as you said, it makes sense that this should be a series of if/else statements. I'll resolve this.